pull_request_template.md

```diff
@@ -1,26 +1,22 @@
 # What this PR is about?
 
-## if it is a new question for the website
+## If it is a new question for the website
 
-> put here the question like: "Tab or Space"
+> Put here the question like: "Tab or Space"
 
 > Upload a screenshot of the question taken from your localhost environment
 
-> Add the label: `new question`
```

Comment:
As external contributors, we cannot set a PR label. Only the maintainer can do it.